plugins/cost-insights/src/utils/styles.ts

```diff
@@ -0,0 +1,455 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createStyles,
+  darken,
+  getLuminance,
+  lighten,
+  makeStyles,
+} from '@material-ui/core';
+import { BackstageTheme } from '@backstage/theme';
+import { CostInsightsTheme, CostInsightsThemeOptions } from '../types';
+
+export const costInsightsLightTheme = {
```

Comment:
These look quite generic, so perhaps worth working into the main theme. It may be a bit of an issue to not have these customizable for orgs that want to change the theme extensively. Perhaps we could experiment with some ways of making these extra themes configurable?